databases/week2/part4.sql

```diff
@@ -0,0 +1,44 @@
+-- Create a new database
+CREATE DATABASE school;
+
+-- Switch to the new database
+USE school;
+
+-- Create the Class table
+CREATE TABLE Class (
+  id INT PRIMARY KEY,
```

Comment:
And again, please, add NOT NULL for ids because later when you're defining foreign keys in other tables, you're adding NOT NULL and it can cause an errors